File: app/src/main/java/org/thoughtcrime/securesms/animation/transitions/CircleToSquareImageViewTransition.java
package org.thoughtcrime.securesms.animation.transitions;

import android.annotation.TargetApi;
import android.content.Context;
import android.util.AttributeSet;

/**
 * Will only transition {@link android.widget.ImageView}s that contain a {@link androidx.core.graphics.drawable.RoundedBitmapDrawable}.
 */
public final class CircleToSquareImageViewTransition extends CircleSquareImageViewTransition {
  public CircleToSquareImageViewTransition(Context context, AttributeSet attrs) {
    super(false);
  }
}

File: app/src/main/java/org/thoughtcrime/securesms/animation/transitions/CircleToSquareImageViewTransition.kt
/*
 * Copyright 2024 Signal Messenger, LLC
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package org.thoughtcrime.securesms.animation.transitions

import android.content.Context
import android.util.AttributeSet

/**
 * Will only transition [android.widget.ImageView]s that contain a [androidx.core.graphics.drawable.RoundedBitmapDrawable].
 */
class CircleToSquareImageViewTransition(
  context: Context,
  attrs: AttributeSet
) : CircleSquareImageViewTransition(false)
